src/virtual-machine.js

```diff
@@ -1071,6 +1071,42 @@ class VirtualMachine extends EventEmitter {
             this.editingTarget.postSpriteInfo(data);
         }
     }
+
+    /**
+     * Set a target variables value if it exists. Return whether it succeeded.
+     * @param {!string} targetId ID of the target which owns the variable.
+     * @param {!string} variableId ID of the variable to set.
+     * @param {!*} value The new value of that variable.
+     * @returns {boolean} whether the target and variable were found and updated.
+     */
+    setVariableValue (targetId, variableId, value) {
+        const target = this.runtime.getTargetById(targetId);
+        if (target) {
+            const variable = target.lookupVariableById(variableId);
+            if (variable) {
+                variable.value = value;
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Set a target variables value if it exists. Return whether it succeeded.
```

Comment:
Same as above if you end up fixing that one.

Reply:
Also it should be 'get' instead of 'set' here and below in the description for the variableId param.